packages/cms-drupal/src/Entity/Channel.php

```diff
@@ -102,6 +102,25 @@ public static function baseFieldDefinitions(EntityTypeInterface $entityType)
                     'type' => 'string_textfield',
                 ])
                 ->setDisplayConfigurable('form', true),
+            'domain' => BaseFieldDefinition::create('string')
+                ->setLabel(new TranslatableMarkup('Domain'))
+                ->setRequired(true)
+                ->setRevisionable(true)
+                ->setDefaultValueCallback(__CLASS__.'::getDefaultKeyValue')
+                ->setDisplayOptions('form', [
+                    'type' => 'string_textfield',
+                ])
+                ->setDescription('The domain name associated to this channel. e.g. www.example.com')
+                ->setDisplayConfigurable('form', true),
+            'locale' => BaseFieldDefinition::create('string')
```

Comment:
Is it to support multilingual? Like https://www.drupal.org/docs/8/multilingual?
Do you want to be able to use the Drupal multilingual way or use a custom one?

Reply:
it is not meant to support a multilingual setup.
it works in isolation. therefore shall I change the name of the field to prevent any conflict ?

Reply:
Ok!
I don't think the naming will be an issue. As far as I know the only things named `locale` are the module in core, a cache tag, and a channel logger.
For multilingual Drupal uses `langcode` or `language` for instance.